### PR TITLE
[#2326] Staging - After un-enrolling the project from CLA Group, Admin is unable to view the Signatures of un-enrolled project.

### PR DESCRIPTION
**Pull request by rinkeshbhutwala**
Fixed to support for project-level permission check.

Signed-off-by: Rinkesh Bhutwala rbhutwal@proximabiz.com

### CODE REVIEW

File: cla-backend-go/v2/signatures/handlers.go [hunk 1]
```diff
@@ -1304,8 +1306,26 @@ func isUserHaveAccessToCLAGroupProjects(ctx context.Context, authUser *auth.User
 		return false
 	}
 	if len(projectCLAGroupModels) == 0 {
-		log.WithFields(f).Debug("no project cla group mappings by CLA Group ID - failed permission check")
-		return false
+		projectCLAGroup, err = projectRepo.GetCLAGroupByID(ctx, claGroupID, false)
+		if err != nil {
+			log.WithFields(f).WithError(err).Warnf("problem loading project cla group mappings by CLA Group ID - failed permission check")
```
Comment (dealako) [inline cla-backend-go/v2/signatures/handlers.go:1311]: Let's update this comment to say: "problem loading cla group by ID - failed permission check"

**Review by dealako — APPROVED** [review]
minor comment

File: cla-backend-go/v2/signatures/handlers.go (hunk 1 above)
Comment (rinkeshbhutwala) [reply cla-backend-go/v2/signatures/handlers.go]: Fixed.

**Review by dealako — APPROVED** [review]
👍

**Pull request merged**